context/.swcrc

```diff
@@ -0,0 +1,28 @@
+{
+    "jsc": {
+        "parser": {
+            "syntax": "ecmascript",
+            "jsx": true,
+            "numericSeparator": false,
+            "classPrivateProperty": false,
+            "privateMethod": false,
+            "classProperty": false,
+            "functionBind": false,
+            "decorators": false,
+            "decoratorsBeforeExport": false
+        },
+        "target": "es2019",
+        "loose": false,
```

Comment:
Is this specific config coming from somewhere?

Reply:
I adapted it from the default config here: https://swc.rs/docs/configuration/swcrc

I needed to remove the `$schema` key because the provided schema did not accept the `experimental` key, which is necessary for styled-components plugin support.